os/kernel/binary_manager/binary_manager.c

```diff
@@ -161,15 +161,15 @@ int binary_manager(int argc, char *argv[])
 	/* Create binary manager message queue */
 	g_binmgr_mq_fd = mq_open(BINMGR_REQUEST_MQ, O_RDWR | O_CREAT, 0666, &attr);
 	if (g_binmgr_mq_fd == (mqd_t)ERROR) {
-		bmdbg("Failed to open message queue\n");
+		printf("[Binary Manager] Failed to open message queue to communicate, errno %d. EXIT!\n", errno);
```

Comment:
1. `to communicate` looks unnecessary. Let's remove.
2. How about this? `[BM] Fail to open ...`?
    1. Binary Manager -> BM
    2. Failed -> Fail